common/src/test/java/com/revenuecat/purchases/backend_integration_tests/LoadShedderBackendIntegrationTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.revenuecat.purchases.backend_integration_tests
+
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.common.networking.Endpoint
+import com.revenuecat.purchases.common.offlineentitlements.ProductEntitlementMapping
+import io.mockk.verify
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+
+class LoadShedderBackendIntegrationTest: BaseBackendIntegrationTest() {
+    override fun apiKey() = Constants.loadShedderApiKey
+
+    @Test
+    fun canPerformProductEntitlementMappingBackendRequest() {
```

Comment:
Can this use the backtick naming?

`can perform product entitlement mapping backend request`